### PR TITLE
Added blueprint check

### PR DESCRIPTION
**Pull request by dafortin**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Added**  blueprint check

### CODE REVIEW

File: blueprints/ember-frost-fields/index.js [hunk 1]
```diff
@@ -1,25 +1,29 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-hook', target: '~1.3.1'}
```
Comment (rox163) [inline blueprints/ember-frost-fields/index.js]: ember hook in this file is ~1.3.1 but in package.json its ^1.4.1. They should probably match up.
i may not have noticed this in the other addons but just spotted it here.

Comment (dafortin) [inline blueprints/ember-frost-fields/index.js]: I will check :) I didn't notice :O

**Comment by rox163** [comment]
:+1:

File: blueprints/ember-frost-fields/index.js [hunk 2]
```diff
@@ -1,25 +1,29 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-hook', target: '^1.4.1'}
```
Comment (sglanzer) [inline blueprints/ember-frost-fields/index.js]: Same question about `hook` coming from `core` in this PR

**Comment by sglanzer** [comment]
Approved

**Pull request merged**